### DISCUSSION TITLE
Requesting if gelbooru could be added as a booru option

### DISCUSSION THREAD

**Discussion by mncho** [end-4/dots-hyprland#623] [Ideas]
I really like the dotfiles, its the best hyprland config I have ever used (and have been using it for a few weeks now, with no issues anymore on nvidia thanks to the new beta drivers :D) However, the only question I have is if you could add booru which isn't yande.re and komachan, namely Gelbooru because it is the one I mainly use as well as the largest. I tried to add it myself but I am not a programmer, and all the edits I tried to make didn't work, so If its not a huge deal, I'd like it to be added. I am guessing it needs some conversion which I am unfortunately not smart enough to do myself, so it would be nice to see it added as an option, thank you!

**Comment by end-4** [comment]
Initially I only had yande.re, but then realized konachan has the same API response format so it was easy to add it
I've looked into alternatives including Gelbooru, but their response formats are different. Since yande.re works I'm just lazy lol.

I'll add this to my todo list though, hopefully I'll get to it some day

**Comment by UMCEKO** [comment]
On it.
